### ISSUE TITLE
Add README file

### ISSUE THREAD

**Issue by vbauer** [rtyley/spongycastle#9]

**Comment by rtyley** [comment]
What would it provide beyond the documentation already at https://rtyley.github.io/spongycastle/ ?

**Comment by vbauer** [comment]
Yes. Actually, I think that tiny README file with this link will be enough. Just to emphasize that documentation exists.

**Comment by mvdan** [comment]
There's already a README, so this should probably be closed :)
